src/itdelatrisu/opsu/GameData.java

```diff
@@ -296,10 +296,6 @@ public HitObjectResult(int time, int result, float x, float y, Color color,
 	/** The difficulty multiplier used in the score formula. */
 	private int difficultyMultiplier = 2;
 
-	/** Beatmap HPDrainRate value. (0:easy ~ 10:hard) */
-	@SuppressWarnings("unused")
-	private float drainRate = 5f;
```

Comment:
This is going to be used in the health formula eventually, so please leave it there!